_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -22,28 +18,41 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger within a [HTML document][].
+The rule applies to any [keyboard event][]: 
+ - where the attribute `key` is a [printable character][] key; and
+ - the method `getModifierState` returns `false`; and
+ - is [dispatched][] to an [event target][] within a [HTML document][].
 
-## Expectation
+## Expectation 1
 
-For the test target, the outcome of at least one of the following rules is passed:
+For each test target:
+ - an [instrument][] to prevent any result of the [keyboard event][] is available; or
+ - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
+ - an [instrument][] is available to force the [keyboard event][] to have the method `getModifierState` return `true`
 
-- [Printable keys only shortcut can be disabled](https://act-rules.github.io/rules/670a30)
-- [Printable keys only shortcut can be remapped](https://act-rules.github.io/rules/aa8b52) 
-- [Printable key shortcut is only active when component has focus](https://act-rules.github.io/rules/1e9941)
+**Note:** The same [instrument][] can be used to disable/force more than one event.
 
-## Assumptions
+**Note:** Preventing results of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
+
+## Expectation 2
+
+For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
 
-This rule assumes as applicable [keyboard shortcuts][keyboard shortcut] those implemented by the [HTML document][] [content][]. Any other means (e.g. browser extensions, browser settings, user agents, external browser applications) are not considered.
+- the [instrument][] is in the same [web page][] of the test target; or
+- instructions for the user to locate the [instrument][] are in the same [web page][] of the test target.
+
+**Note:** If the set of instruments has more than one instrument, not every instrument of the set needs to be in the same location.
```

Comment:
Probably we could:

* Move the idea of instrument or set of instruments to expectation 1.
* Remove references to the fact that they don't need to be in the same location. it is not clear to me where this is taken from.

Reply:
I'll update the rule, and hopefully it is easier to understand now.